src/supervisor/requirements.txt

```
@@ -10,4 +10,7 @@ graphviz
 pymannkendall
 scikit-learn
 seaborn
+reportlab
+PyPDF2
+ruptures
```

I was missing statsmodels
```suggestion
ruptures
statsmodels
```

---

Thanks @smcavey . Fixed now.